src/main/resources/bidder-config/aax.yaml

```diff
@@ -0,0 +1,21 @@
+adapters:
+  aax:
+    endpoint: https://prebid.aaxads.com/rtb/pb/aax-prebid?src={{PREBID_SERVER_ENDPOINT}}
+    meta-info:
+      maintainer-email: product@aax.media
+      app-media-types:
+        - banner
+        - video
```

Comment:
Here https://github.com/prebid/prebid.github.io/pull/3696/files video is not mentioned. Is it docs mistake or is it configuration mistake(or maybe i missed something)?

Reply:
We have added video in prebid doc